src/Layouts/Partials/Layer.vala

```diff
@@ -83,24 +85,23 @@ public class Akira.Layouts.Partials.Layer : Gtk.ListBoxRow {
 
     // public Akira.Shape shape { get; construct; }
```

Comment:
I think this is an old leftover, it can be removed.

Reply:
Ok!